common/lib/common-utils/src/test/types/validateCommonUtilsPrevious.generated.ts

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+
```

Comment:
New build-tools/build-cli generates updated type tests files. All the changes here seem expected to me.